docs/rhoas/cli-context/README.adoc

```diff
@@ -0,0 +1,240 @@
+$ rhoas context set-kafka
+----
+--
+
+. Setting Service Registry Instance in the current context
++
+--
+.Set Service Registry instance in the current context
+[source,shell]
+----
+$ rhoas context set-service-registry
+----
+
+[NOTE]
+====
+Creating a new service instance automatically sets it in the current context.
+====
+--
+
+. Check status of the current context
++
+--
+After setting service(s) in the current context, their staus can be checked using the `rhoas context status` command
+.View status of the current context
+[source,shell]
+----
+$ rhoas context status
+----
+--
+
+[id="proc-generating-config-cli_{context}"]
+== Generating application configurations for contexts
+
+`rhoas generate-config` command generates application configurations that can be used to connect to the services set in context.
+
+.Generating configurations for context named as my "my-context"
+[source,shell]
+----
+$ rhoas generate-config --name my-context --type json
+----
+
+=== Generating configurations for quarkus applications
+
+[id="proc-importing-quarkus-sample-code_{context}"]
+==== Importing the Quarkus sample code
+
+[role="_abstract"]
+For this quick start, you'll use the Quarkus sample code from the {product-kafka} {samples-git-repo}[Guides and Samples^] repository in GitHub.
+After you understand the concepts and tasks in this quick start, you can use your own Quarkus applications with {product-kafka} in the same way.
+
+.Procedure
+. On the command line, clone the {product-kafka} {samples-git-repo}[Guides and Samples^] repository from GitHub.
++
+.Cloning the guides and samples repository
+[source,subs="+attributes"]
+----
+git clone {samples-git-repo} app-services-guides
+----
+. In your IDE, open the `code-examples/quarkus-kafka-quickstart` directory from the repository that you cloned.
+. Generate the application configurations for the quarkus application.
++
+--
+[source,shell]
+----
+$ rhoas generate-config --type env --output-file .env 
+----
+--
+. Create the "prices" Kafka topic.
++
+--
+[source,shell]
+----
+$ rhoas kafka topic create --name prices
+----
+--
+. Create ACLs for the service account created using the `rhoas generate-config` command.
++
+--
+[source,shell]
+----
+$ rhoas kafka acl grant-access --producer --consumer --service-account <client_id> --topic prices --group all
+----
+--
+. Run the Quarkus application in dev mode.
++
+--
+[source,shell]
+----
+./mvnw quarkus:dev
+----
+--
+
+[role="_abstract"]
+The Quarkus application starts produces and consumes messages to and from the "prices" Kafka topic.
+After the application is running, in a web browser, go to http://localhost:8080/prices.html[^] and verify that the `Last price` is updated.
+
+If the Quarkus application fails to run, review the error log in the terminal and address any problems. Also review the steps to ensure that the Quarkus application and Kafka topic are configured correctly.
+
+[id="proc-switching-context-cli_{context}"]
+== Switching between contexts
```

Comment:
At this point we have satisfied our main user goal. Switching user context should be more documentation/reference than content of the guide IMHO. It is too obvious and boring from my take

Reply:
Everything bellow doesn't seem to match any user experience we wanted to flag with this guide